src/components/SearchInput.tsx

```diff
@@ -0,0 +1,38 @@
+import { Formik, Form, Field } from "formik";
+import { Link, useNavigate } from "react-router-dom";
+
+type searchtype = {
```

Comment:
what is difference between type and interface read about it and I'll take a quiz about this and also learn about filter, find, map, foreach and includes functions